src/Microsoft.AspNetCore.Server.Testing/Deployers/IISExpressDeployer.cs

```diff
@@ -34,21 +34,23 @@ public override DeploymentResult Deploy()
                 DotnetPublish();
             }
 
+            var webRootLocation = DeploymentParameters.PublishApplicationBeforeDeployment ? DeploymentParameters.PublishedApplicationRootPath : DeploymentParameters.ApplicationPath;
```

Comment:
To be consistent with what we used as hosting keys, we should call it `contentRoot`